README.md

```diff
@@ -219,7 +218,7 @@ If a HOWL stream starts with blank lines, these will be parsed as blank blocks.
 
 This blank block:
 
-        
```

Comment:
Grr. That's an `after-save` hook I've got on my local that deletes trailing whitespace. I'll put a special case in it for this file (and revert this before merging, obviously).